public/index.html

```diff
@@ -1,15 +1,36 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>WDI Radio</title>
-    <script src="js/app.js"></script>
+<html data-ng-app="wdiRadio">
+<head>
 
-  </head>
-  <body>
-    <header>WDI Radio</header>
+  <title>WDI Radio</title>
 
-    Welcome to the World's Best Song App
+  <script src="https://www.gstatic.com/firebasejs/3.2.0/firebase.js"></script>
+  <script>
+  // Initialize Firebase
+  var config = {
+    apiKey: "AIzaSyBkrkA47hBkEJyE-2Qz2GVaqKiP0cUN0oY",
```

Comment:
It's a good idea to not obscure your api key and would advise not pushing it to github